desktop_projects/desktop_sim/tests/test_gps_format_conversion.cpp

```diff
@@ -0,0 +1,19 @@
+#include <gtest/gtest.h>
+
+#include "math_utils.h"
+
+TEST(GpsFormatConversion, SimplePositive) {
+  ASSERT_FLOAT_EQ(decimalminutes_to_decimaldegrees(4315.0), 43.25);

```

Comment:
nit: is there any way to use fewer magic numbers here? at a glance idk why i would expect this test to pass. A comment saying "this is how you go from ddmm.mmmm to ddd.ddddd" would probably do it